src/main/pages/portable-workspaces/factories-getting-started.adoc

```diff
@@ -8,62 +8,69 @@ folder: portable-workspaces
 ---
 
 
-A Factory is a template containing configuration to automate generating new Workspace with factory identifier added to IDE URL. Factories can be used to create replicas of existing workspaces or to automate the provisioning of statically or dynamically defined workspaces.
+A factory is a template containing configuration to automate generating new Workspace with factory identifier added to the IDE URL. Factories can be used to create replicas of existing workspaces or to automate the provisioning of statically or dynamically defined workspaces.
 
 [id="try-a-factory"]
-== Try a Factory
+== Trying a factory
 
-Clone a public workspace on `che.openshift.io` by clicking the button below. https://che.openshift.io/f?id=factorymtyoro1y0qt8tq2j[image:https://che.openshift.io/factory/resources/factory-contribute.svg[Try a Factory]]
+To clone a public workspace on `che.openshift.io`, click the button: https://che.openshift.io/f?id=factorymtyoro1y0qt8tq2j[image:https://che.openshift.io/factory/resources/factory-contribute.svg[Try a Factory]].
```

Comment:
There are a lot of conditionals in this document switching based on what's being documented, but not here. Is that unnecessary in this case?